client/src/pages/Home.js

```diff
@@ -6,91 +6,109 @@ import ls from "local-storage";
 import "../sass/Home.scss";
 
 const Home = (props) => {
-    const [events, setEvents] = useState([]);
-    const [isLoading, setIsLoading] = useState(false);
-    const [event, setEvent] = useState("--SELECT ONE--");
+  const [events, setEvents] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [event, setEvent] = useState("--SELECT ONE--");
 
-    async function fetchEvents() {
-        try {
-            setIsLoading(true);
-            const res = await fetch("/api/events?checkInReady=true");
-            const resJson = await res.json();
-            await resJson.unshift("--SELECT ONE--");
+  async function fetchEvents() {
+    try {
+      setIsLoading(true);
+      const res = await fetch("/api/events?checkInReady=true");
+      const resJson = await res.json();
+      await resJson.unshift("--SELECT ONE--");
 
-            setEvents(resJson);
-            setIsLoading(false);
-        } catch (error) {
-            console.log(error);
-            setIsLoading(false);
-            // setIsError(error);
-            // alert(error);
-        }
+      setEvents(resJson);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+      setIsLoading(false);
+      // setIsError(error);
+      // alert(error);
     }
+  }
 
-    const handleEventChange = (e) => {
-        setEvent(e.currentTarget.value);
-        // setIsQuestionAnswered(true);
-    };
+  const handleEventChange = (e) => {
+    setEvent(e.currentTarget.value);
 
-    useEffect(() => {
-        fetchEvents();
+    // setIsQuestionAnswered(true);
+  };
+  const slackButton = () => {
+    console.log("SLACK BUTTON");
+    fetch("api/slack/event?checkInReady=true");
```

Comment:
Do we need to hardcode checkInReady on all API calls?

Reply:
@nickbeaird This is a good question but it's better to address it to the legacy codebase. Please create a task for fixing this API endpoint and further client cleanup with a clear description of what should be done.
@Jonathanko52 skip this comment.

Reply:
@alex-anakin, this is a new method, so it is not legacy code. I defer to you on this one though.